pkg/skaffold/config/util.go

```diff
@@ -372,6 +373,34 @@ func UpdateHaTSSurveyTaken(configFile string) error {
 	return err
 }
 
+func UpdateUserSurveyTaken(configFile string, id string) error {
+	ai := fmt.Sprintf(updateUserTaken, id)
+	aiErr := fmt.Errorf("could not automatically update the survey prompted timestamp - please run `%s`", ai)
+	configFile, err := ResolveConfigFile(configFile)
+	if err != nil {
+		return aiErr
+	}
+	fullConfig, err := ReadConfigFile(configFile)
+	if err != nil {
+		return aiErr
+	}
+	if fullConfig.Global == nil {
+		fullConfig.Global = &ContextConfig{}
+	}
+	if fullConfig.Global.Survey == nil {
+		fullConfig.Global.Survey = &SurveyConfig{}
+	}
+	for _, s := range fullConfig.Global.Survey.UserSurveys {
+		if s.ID == id {
+			s.Taken = util.BoolPtr(true)
+			return nil
+		}
+	}
+	fullConfig.Global.Survey.UserSurveys = append(fullConfig.Global.Survey.UserSurveys,
+		&UserSurvey{ID: id, Taken: util.BoolPtr(true)})
+	return nil
```

Comment:
We need to write this file out, right?
```suggestion
	err = WriteFullConfig(configFile, fullConfig)
	if err != nil {
		return aiErr
	}
	return nil
```

Reply:
Thanks for catching this!